src/app/charts/box-plot/pbox-plot-view/pbox-plot-view.component.ts

```diff
@@ -133,6 +133,12 @@ export class PBoxPlotViewComponent implements OnInit, OnDestroy, AfterViewInit {
         this.chartService.removeChart(this.chart);
     }
 
+    getGeneSymbol(): string {
+        const geneSymbol = this.geneService.getCurrentInfo().hgnc_symbol ?
```

Comment:
I think this could be changed to:
const geneSymbol = this.geneService.getCurrentInfo().hgnc_symbol ?? this.geneService.getCurrentInfo().ensembl_gene_id;

Reply:
oh that's convenient!